.travis.yml

```diff
@@ -98,6 +98,8 @@ install:
 script: |
   set -e
   if [[ $JOB = docs ]]; then
+    # install docs dependencies
+    pip install '.[docs]'
```

Comment:
can I just extend line 86?
I'm a little confused why this isnt `pip install mock` instead.